Package.swift

```diff
@@ -2,7 +2,7 @@
 import PackageDescription
 
 let package = Package(
-    name: "swift-baggage-context",
+    name: "swift-context",
     products: [
         .library(
             name: "Baggage",
```

Comment:
This module exists only for users which do NOT want to depend on SwiftLog for some reason.

Reply:
@lukasa -- will you be ok with such package structure in order for nio to depend only on the `Baggage` here?